app/backend/src/clj/user.clj

```diff
@@ -230,37 +231,15 @@
 ;;
 ;; Commands and queries from the REPL
 ;;
-(def logged-in-user-id (atom nil))
-
-(defn local-login [user-id]
-  (reset! logged-in-user-id user-id)
-  (log/info "Locally logged in as " user-id))
-
-(defn action-ctx [user-id]
-  (let [db (db)]
-    {:conn (db-connection)
-     :db db
-     :user (user-db/user-info db user-id)
-     :session "foo"}))
-
-(defn local-query
-  ([query args]
-   (if-let [user-id @logged-in-user-id]
-     (local-query user-id query args)
-     (log/error "Not logged in! Call user/local-login with an existing user id to log in.")))
-
-  ([user-id query args]
-   (db-api-handlers/raw-query-handler (action-ctx user-id)
-                                      {:args args
-                                       :query query})))
-
-(defn local-command
-  ([command args]
-   (if-let [user-id @logged-in-user-id]
-     (local-command user-id command args)
-     (log/error "Not logged in! Call user/local-login with an existing user id to log in.")))
-
-  ([user-id command args]
-   (db-api-handlers/raw-command-handler (action-ctx user-id)
-                                        {:payload args
-                                         :command command})))
+(intern 'user
```

Comment:
why intern instead of `def`?

Reply:
Yeah true, since they're not macros, might as well just use a def.